.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
```

Comment:
Please don't change this file :)

Reply:
The easiest is probably to copy the content of this file from Mate's main layout_html-form repository and commit it immediately. Even if it is changed on your machine during development, just don't commit it again ;)